xbmc/rendering/dx/DeviceResources.h

```diff
@@ -118,6 +118,7 @@ namespace DX
     // Gets debug info from swapchain
     DEBUG_INFO_RENDER GetDebugInfo() const;
     std::vector<DXGI_COLOR_SPACE_TYPE> GetSwapChainColorSpaces() const;
+    bool SetMultithreadProtected(bool enabled);
```

Comment:
```suggestion
    bool SetMultithreadProtected(bool enabled) const;
```

Reply:
I didn't because the state of DeviceResources changes from an external point of view, but technically the compiler will accept it
Are you sure?

Reply:
If compiler accepts it then is right. In fact is not changing any member variable.

Reply:
OK. This is not a strong example with a class that can barely be called a wrapper but please keep this in mind for future
http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rconst-fct
https://isocpp.org/wiki/faq/const-correctness#logical-vs-physical-state
https://isocpp.org/wiki/faq/const-correctness#logical-vs-physical-const